torchdynamo/bytecode_analysis.py

```diff
@@ -159,5 +159,5 @@ def stacksize_analysis(instructions):
         return low + 32
 
     assert fixed_point.value, "failed to reach fixed point"
-    assert low >= 0
+    # assert low >= 0
```

Comment:
Line 162 doesn't pass 5 unit tests and line 161 breaks ```maml``` model in torchbench. I'm working on debug this, would be happy if you have any suggestion. @jansel

Reply:
Usually this error means we generated invalid bytecode.  There is a commented out print earlier in the same function I've used to debug that in the past.

Reply:
The print out helper finally helped me find & fix the bug, which is because I got messed up between comment & comment out of adding ```RERAISE``` to ```TERMINAL_OPCODES``` as I have to test with Python 3.7/3.8/3.9 at the same time. Now the code is in good shape and passed all unit tests & benchmark models.

Reply:
Great!   Glad we caught that one.